conda-forge.yml

```diff
@@ -1,6 +1,10 @@
-github:
-  branch_name: main
-  tooling_branch_name: main
+azure:
+  free_disk_space: true
+  store_build_artifacts: true
```

Comment:
Would just remember to revert this and re-render after testing is done

Reply:
Yeah was more meaning dropping the `store_build_artifacts` field

Reply:
Oh! I kinda like using that so I can test the builds, does it chew through a lot of resources?

Reply:
It can yeah. Generally it is better to enable for debugging and then turn off. It's pretty simple to readd when needed though

Reply:
Good to know, thanks!